Library/Homebrew/cmd/missing.rb

```diff
@@ -6,15 +6,41 @@ module Homebrew
   def missing
     return unless HOMEBREW_CELLAR.exist?
 
-    ff = if ARGV.named.empty?
+    formulae = if ARGV.named.empty?
       Formula.installed
     else
       ARGV.resolved_formulae
     end
+    if ARGV.json == "v1"
+      missing = print_missing_json(formulae)
+    else
+      missing = print_missing(formulae)
```

Comment:
On second thought, maybe it shouldn't be changed in `outdated` because that would be a purely stylistic change. Here, on the other hand, we're talking about new code and that should be written with the currently preferred style in mind.